### PR TITLE
Add shortDescs to moves

### PR DESCRIPTION
**Pull request by AndrewGoncel**

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -181,6 +183,7 @@ exports.BattleMovedex = {
 	nextlevelstrats: {
 		accuracy: true,
 		category: "Status",
+		shortDesc: "Boosts max level in 5",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:186]: by 5

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -363,6 +367,7 @@ exports.BattleMovedex = {
 		accuracy: 85,
 		basePower: 100,
 		category: "Special",
+		shortsDesc: " Partial trapping move, persists even when user is not in play.",
 		id: "maelstrm",
```
Comment (DeathlyPlays) [inline mods/seasonal/moves.js:371]: shortDesc not shortsDesc might want to delete the space before Partial also

File: mods/seasonal/moves.js [hunk 3]
```diff
@@ -620,7 +626,7 @@ exports.BattleMovedex = {
 		pp: 10,
 		priority: 0,
 		category: "Special",
-		shortDesc: "For 5 turns, negates all Ground immunities. More power the heavier the target.",
+		shortDesc: "Adds Trick Room. Deals damage.",
```
Comment (DeathlyPlays) [inline mods/seasonal/moves.js:629]: Might want to include how its weight dependent (more damage the heavier the user is than the target)

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: mods/seasonal/moves.js (hunk 3 above)
Comment (HoeenCoder) [reply mods/seasonal/moves.js:629]: No, short desc needs to be very short do it fits within ``/data``

Comment (DeathlyPlays) [reply mods/seasonal/moves.js:629]: Fair enough but perhaps something like "Adds Trick Room; more power heavier the user."

Comment (HoeenCoder) [reply mods/seasonal/moves.js:629]: that works

**Comment by DeathlyPlays** [comment]
Bump @AndrewGoncel

**Pull request closed without merging**